src/components/ViewClub.js

```diff
@@ -0,0 +1,361 @@
+/* eslint-disable no-unused-vars */
```

Comment:
this doesn't seem like a component to me. A component is reusable and can be isolated. This is more like a view to me so we should put it under something like /src/views

Reply:
I think this is just as much a component as anything else in our app. It contains a form and handles state like many of our other components.